locales/de-DE.json

```diff
@@ -7,9 +7,9 @@
     "route_loaded": "Seite {0} geladen"
   },
   "account": {
-    "avatar_description": "{0}'s Avatar",
+    "avatar_description": "{0}s Avatar",
     "blocked_by": "Du wurdest von diesem Account geblockt",
-    "blocked_domains": "Gesperrte Domänen",
+    "blocked_domains": "Gesperrte Domains",
```

Comment:
This is also not really correct

Domänen ist denke ich die richtige Übersetzung

![image](https://user-images.githubusercontent.com/7195563/216431935-48aed33d-74a1-43e2-bae1-a3a4f8a2acb6.png)

Reply:
Man sperrt hier aber keine Domänen, sondern das wird bei Domains ausgespielt also domain.tld

Reply:
> Man sperrt hier aber keine Domänen, sondern das wird bei Domains ausgespielt also domain.tld

Versteh ich leider nicht so ganz, bitte versuch nochmal genauer zu erklären

Reply:
Bei dem Feature geht es ja darum, Instanzen zu sperren. Also zum Beispiel Mastodon.social.

Reply:
Wie wäre es dann mit

```suggestion
    "blocked_domains": "Gesperrte Mastodon Instanzen",
```

Reply:
Gesperrte Mastodon-Instanzen ;-)

Reply:
```suggestion
    "blocked_domains": "Gesperrte Mastodon-Instanzen",
```

Reply:
That isn't quite right nor true.
When you block an instance you more or less block the domain because a domain IS an instance. And afaik do you block the domain as a whole to not deal with any new sub-domains trying to bypass block lists.

So blocking a domain is technically correct here.